source/visionEnhancementProviders/screenCurtain.py

```diff
@@ -74,14 +74,15 @@ class Magnification:
 			_MagGetFullscreenColorEffectArgTypes
 		)
 		MagGetFullscreenColorEffect.errcheck = _errCheck
+		MagShowSystemCursor = _MagShowSystemCursorFuncType(
+			("MagShowSystemCursor", _magnification),
+			_MagShowSystemCursorArgTypes
+		)
+		MagShowSystemCursor.errcheck = _errCheck

```

Comment:
Could you add comments to this try block please. Something along the lines of `These magnification functions are not available on prior to Windows 8, and will raise an AttributeError`